README.md

```diff
@@ -1,5 +1,7 @@
 # DATAPROC-MULTICLUSTERS
 
+⚠️ Starting with DSS version 14 this plugin is considered \"deprecated\" (see [deprecation notice](https://doc.dataiku.com/dss/latest/hadoop/dynamic-dataproc.html))
```

Comment:
I don't see why `"` need to be escaped, but looks like it renders just fine

Reply:
copy passte from the json, will remove it, thanks!